recipes/facexlib/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "facexlib" %}
+{% set version = "0.2.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/facexlib-{{ version }}.tar.gz
+  sha256: 31e20cc4ed5d63562d380e4564bae14ac0d5d1899a079bad87621e13564567e4
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  host:
+    - python
+    - pip
+    - cython
+    - numpy
```

Comment:
This [package lists](https://github.com/xinntao/facexlib/blob/master/setup.py#L105) cython and numpy but I cannot see a reason for it. Looks like it can be noarch.

Reply:
They list it in their `setup_require` https://github.com/xinntao/facexlib/blob/master/setup.py#L105

Reply:
You are correct but they are not really using it as far as I can see.

Reply:
@ocefpaf It worked - thanks for the suggestion